src/jit/instr.h

```diff
@@ -288,6 +288,7 @@ enum InstructionSet
 {
 #ifdef _TARGET_XARCH_
     InstructionSet_SSE2,
+    InstructionSet_SSE3_4,
     InstructionSet_AVX,
```

Comment:
I think the entries in this enum need comments to specifically describe what instruction sets are referred to for each